wesnoth-help.po

```diff
@@ -6636,6 +6628,70 @@ msgid ""
 "eating carrion and even committing cannibalism, which are both regular "
 "occurrences."
 msgstr ""
+"Ještěráci jsou ještěrkovití tvorové. Jsou menší a štíhlejší než lidé a "
+"zřídka měří víc než desetileté dítě. Nicméně od špičky nozder po konec ocasu "
+"může být ještěrák tak dlouhý, jak je průměrný muž vysoký. Ještěráčtí "
+"válečníci jsou lehcí a obratní a nejraději bojují stejnou taktikou, jako při "
+"lovu - proklouzávají nepřátelskými liniemi a zaměřují se na slabé a zraněné, "
+"zatímco se vyhýbají útočníkům.\n"
+"\n"
+"<header> text = 'Společnost' </header>\n"
+"Ještěráci jsou velmi tajemná stvoření, což plyne z jejich tendence žít v "
+"oblastech nehostinných pro ostatní rasy, jako jsou třeba bažiny. Jejich "
+"myšlení je extrémně fatalistické, ještěráci věří, že všechny události v "
+"životě lze předpovědět pomocí složitých astronomických výkladů.\n"
+"\n"
+"Ještěrácká kultura je ostře pohlavně segregovaná. V rámci každého pohlaví "
+"mezi sebou jeho zástupci soupeří a prostřednictvím svých dovedností, "
+"odhodlání a reputace tak vytvářejí jasný hierarchický řád, který je vedený "
+"svým vedoucím. V případech, kdy obě pohlaví interagují, nebojují vzájemně o "
+"nadvládu. Místo toho situaci určuje dominantní pohlaví. Vedoucí samců je "
+"alfa ve vesnici nebo táboře klanu, zatímco vedoucí samic je dominantní "
+"kdekoli jinde. Tak to pokračuje v sestupné struktuře, přičemž každý samec "
+"nebo samice je dominantní nad jakýmkoli členem opačného pohlaví s nižším "
+"postavením a podrobuje se členům opačného pohlaví s postavením vyšším.\n"
+"\n"
+"Segregace a střídání genderové dominance ještěrácké společnosti je výsledkem "
+"jejich jasně definovaných genderových rolí. Samice mají odpovědnost za "
+"lovení a nacházení jídla, což jsou dovednosti, které je nakonec vycvičí, aby "
+"se staly válečnicemi: rváčkami, ochránkyněmi a přepadávačkami, kterých se "
+"ostatní rasy tak bojí. Samci jsou mezitím zodpovědní za hlídání snůšky - "
+"vajec zanechaných samicemi. I když jim to ponechává čas na rozvoj a "
+"zdokonalování umění astrologie, léčení a magie, vystavuje je to také "
+"značnému nebezpečí, protože se tím stávají stabilními cíli nepřátel klanu "
+"číslo jedna - ostatních ještěráckých klanů.\n"
+"\n"
+"Nové ještěrácké klany jsou zakládány, když jsou vyčtena správná astrologické "
+"znamení. Každá samice označená hvězdnou konjukcí je provolána ploditelkou, "
+"vybere si skupinu jedinců s nižším postavením a opustí svůj původní klan. "
+"Často jsou označeny všechny samice se specifickou vlastností, což způsobí, "
+"že se „vylíhne“ více nových klanů najednou. Výběr jedinců se řídí "
+"jednoduchým procesem: žádná skupina, která opouští klan, nemůže být větší "
+"než kterákoli jiná, všechny odchozí skupiny nemohou dohromady být větší než "
+"zbývající klan a samice s vyšším postavením si může přetáhnout jedince z "
+"výběru jiné, méně privilegované samice.\n"
+"\n"
+"V důsledku rychlého populačnímu růstu, častým rozkolům v klanech a "
```

Comment:
```suggestion
"Díky rychlému populačnímu růstu, častým rozkolům v klanech a "
```

Reply:
Opraveno skloňování, to díky by mi tam nesedělo.